### DISCUSSION TITLE
Build Order Tool v1.0

### DISCUSSION THREAD

**Discussion by LENpolygon** [LENpolygon/Build-Order-Tool-AoE4-#31] [Announcements]
Another big update!

Comparing to last time updates include:

- Users can sign up
- Users can login
- Users can logout
- Logged in users have their nickname added as build author "BUILD (by [AUTHOR])"
- Logged in users can upvote / downvote builds allowing for some community moderating
- Logged in users can delete their own builds. This also allows logged in users to "edit" their builds by adjusting and uploading a new one, then deleting the old.
- Updated popularity scoring algorithm
- Cleaned up some code here and there
- Made a spaghetti mess in some other places
- Increased the version counter to v1.0.0

Still much room for improvements. See open issue list. Ideas and suggestions welcome.

<hr /><em>This discussion was created from the release <a href='https://github.com/LENpolygon/Build-Order-Tool-AoE4-/releases/tag/v1.0'>Build Order Tool v1.0</a>.</em>

**Comment by jensbuehl** [comment]
Thanks for the effort that you have put into that!
There is a lot of potential, especially together with the overlay tool. :)

Suggestions and Questions
- Improve UI on home screen, e.g. show info per build (e.g. rating percentage, build type,...), no more 2 column layout to free some space for the important info. (from a user's perspective)
- Allow for more meta info per build order: e.g. video link, description field, build type (rush, land boom, water boom,...)
- Is there an API in place to query the builds per user? This would allow for a closer integration with Overlay Tools so that the user can maintain builds in a single place and automatically pull it into the overlay. (instead of copy/paste)
- Obviously some UX stuff (e.g. management of user builds is not ideal, yet)

**Reply by LENpolygon** [reply]
I'm glad that you like the tool and I feel like all your ideas are great suggestions! 
At the moment my focus is more on other projects. I've put a lot of time in this tool in the past year and now it's time to focus more on others.

Off course I will be able to assist with updates and make sure code merges happen!
